jest.config.js

```diff
@@ -2,4 +2,19 @@
 module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'node',
-};
\ No newline at end of file
+  /**
+   * Only collect coverage for ChallengeManager and utilities used by ChallengeManager
+   * merkle.ts IS a utility used by challenge manager. But this utility is fairly specific
+   *  to merkle-tools dependency. So it is ignored for now.
+   */
+  collectCoverage: true,
+  collectCoverageFrom: ['src/**/*.{js,ts}', '!src/{index,merkle,auto-disputer}.ts'],
```

Comment:
Beyond the scope of this PR but `auto-disputer` probably deserves to get it's own set of tests and get included in the coverage results.

Reply:
I think that's a fair point! Since the challenge manager will be off-chain code, we can be more lenient with our code coverage requirement.